plugins/themes/default/styles/articleRevisions.less

```diff
@@ -0,0 +1,32 @@
+/**
+ * @file lib/pkp/styles/articleRevisions.less
```

Comment:
This needs a little work to get it standardized alongside the rest of our Javascript, which gets linted, optionally compiled and minified, etc.